src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js

```diff
@@ -784,3 +791,17 @@ export const initApplicationSubmitMock = token => {
     },
   });
 };
+
+export const initValidVaFileNumberMock = token => {
```

Comment:
This is the only real addition in this pull request. It mocks the call for checking if a user has a valid va file number. Most of the changes are just rearranging the order of form chapters.